src/richie/apps/core/context_processors.py

```diff
@@ -124,6 +125,18 @@ class WebAnalyticsContextProcessor:
     frontend.
     """
 
+    # pylint: disable=no-self-use
+    def frontend_context_processor(self, request: HttpRequest) -> dict:
+        """
+        Additional web analytics information for the frontend react
+        """
+        context = {}
+        if getattr(settings, "WEB_ANALYTICS_ID"):
+            context["web_analytics_provider"] = getattr(
+                settings, "WEB_ANALYTICS_PROVIDER", "google_analytics"
```

Comment:
I'll do not use `google_analytics` as a default value. I think it should be explicitly declared through settings. WDYT?